roles/confluent.common/vars/main.yml

```diff
@@ -10,3 +10,7 @@ confluent_cli_goarch:
 
 ### Filename of Confluent CLI binary inside the archive
 confluent_cli_binary: confluent
+
+redhat_supported_versions: ['7', '8']
+ubuntu_supported_versions: ['16', '18']
+debian_supported_versions: ['9']
```

Comment:
I was thinking about variables on the tasks. Sorry for not being precise. Role variables seems like a bit overkill...

Reply:
yep haha good thinking!